install/0000_80_machine-config-operator_01_containerruntimeconfig.crd.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: containerruntimeconfigs.machineconfiguration.openshift.io
```

Comment:
do we have this in openshift/api ?

Reply:
same question as for those below. All `*.openshift.io` APIs belong into that repo. Otherwise, we are unable to do cross-component API tasks in a consistent way and have no API review either.

Reply:
don't think so. Doing quick grep of "containerruntimeconfigs" in openshift/api repo gives no result.

Reply:
Agree and while working on this PR I was thinking of doing a spike in 4.9 and work on moving MCO APIs to openshift/api repo.